esp-hal-common/src/mcpwm/timer.rs

```diff
@@ -1,3 +1,11 @@
+//! MCPWM peripheral - timer module
+//!
+//! ## Overview
+//! The `timer` module is a part of the `MCPWM (Motor Control Pulse Width
+//! Modulator)` driver for ESP chips. It provides an interface to configure and
+//! use timers for generating `PWM` signals used in motor control and other
+//! applications.
+
```

Comment:
If you're at it, could you please fix the typo in line 38? :D `prescalar` -> `prescaler`.